js/components/closed-caption-multi-audio-menu/closedCaptionMultiAudioMenu.js

```diff
@@ -0,0 +1,128 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+var Utils = require('../utils');
+var classnames = require('classnames');
+var CONSTANTS = require('../../constants/constants');
+var Tab = require('./tab');
+var MultiAudioTab = require('./multiAudioTab');
+
+var ClosedCaptionMultiAudioMenu = React.createClass({
+  componentDidMount: function() {
+    var multiAudioCol = ReactDOM.findDOMNode(this.refs.multiAudioCol);
+    var closeCaptionsCol = ReactDOM.findDOMNode(this.refs.closeCaptionsCol);
```

Comment:
I see closeCaptions instead of closedCaptions in this file. Can we change all these vars/references to closedCaptions? Closed captions is the standard name. This will help maintenance and debugging when searching for closed captions strings, for example.

Reply:
Good catch, thanks!

Reply:
\+